src/store/src/DistanceStrategyEnum.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\AI\Store;
+
+/**
+ * @author Guillaume Loulier <personal@guillaumeloulier.fr>
+ */
+enum DistanceStrategyEnum: string
```

Comment:
Doesn't need the suffix if you ask me :)

```suggestion
enum DistanceStrategy: string
```